src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -735,9 +738,17 @@ bool Dataloader::_load_map_dir(GameManager& game_manager) const {
 		ret = false;
 	}
 
-	if (!map.load_region_file(parse_defines(lookup_file(append_string_views(map_directory, region))).get_file_node())) {
-		Logger::error("Failed to load region file!");
-		ret = false;
+	{
+		std::vector<colour_t> colours;
+		if (!map.load_region_colours(parse_defines(lookup_file(append_string_views(interface_directory, region_colours))).get_file_node(), colours)) {
+			Logger::error("Failed to load region colours file!");
+			ret = false;
+		}
+
```

Comment:
```suggestion
		using namespace OpenVic::colour_literals;
		colours.push_back(0xFFFFFF_rgb);
```
White needs to be added to the end of the list of colours. It can be created from the integer literal `0xFFFFFF` with the suffix `_colour`, athough I'd prefer if the suffix was `_rgb`, to match `_argb`. Can you edit [this line in Colour.hpp](https://github.com/OpenVicProject/OpenVic-Simulation/blob/master/src/openvic-simulation/types/Colour.hpp#L402) to make it `_rgb` instead of `_colour`.